### PR TITLE
Simplify URLs

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Fixes https://github.com/cds-snc/notification-api/issues/1008 + https://github.com/cds-snc/notification-api/issues/979

Removed the `/templates` namespace as described in the issue, moved the contact form to `/contact` and removed templates + old redirections that were not used by Notify.

I've added redirections for previous URLs to avoid 404s

### CODE REVIEW

**Comment by AntoineAugusti** [comment]
> This pull request **introduces 1 alert** when merging [8e62a2b](https://github.com/cds-snc/notification-admin/commit/8e62a2bc307bac6172e663ccbe648edacfb3bd3c) into [7b7eb77](https://github.com/cds-snc/notification-admin/commit/7b7eb7729f6d78ea04a73960356c64008f3d7662) - [view on LGTM.com](https://lgtm.com/projects/g/cds-snc/notification-admin/rev/pr-542622f0d74694d3a556e58cc0239c5ffc36599f)
> 
> **new alerts:**
> 
> * 1 for Potentially unsafe external link

Reviewed and the link is on our website, pointing to our own contact page.

**Comment by AntoineAugusti** [comment]
Fixed https://github.com/cds-snc/notification-api/issues/979 in https://github.com/cds-snc/notification-admin/pull/763/commits/fc7cf5fa0fa970ffc0bf1eb4f086a1b6447c3c83 @anikbrazeau

**Comment by AntoineAugusti** [comment]
@brdunfield Reverting to draft this PR. I've made changes to the contact page to fix the bug that you fixed recently in https://github.com/cds-snc/notification-admin/pull/765.

I'm working on adding tests to the contact form + contact page at the moment to prevent regressions in the future.

**Comment by AntoineAugusti** [comment]
@brdunfield Thanks for bearing with me! I've added a bunch of tests for the contact form in `tests/app/main/test_contact.py`, https://github.com/cds-snc/notification-admin/pull/763/commits/1ccf023f724ac97e656d33eed1c994bd38000d6e

File: app/main/views/index.py [hunk 1]
```diff
@@ -43,14 +41,14 @@ def index():
     form = ContactNotifyTeam()
 
     # catch with the honeypot field
-    if(form.phone.data):
-        return redirect(url_for('.feedback', ticket_type='thanks'))
+    if (form.phone.data):
+        abort(403)
```
Comment (brdunfield) [inline app/main/views/index.py:45]: What's the reason for this change?

File: a11y.json [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
-    "support/ask-question-give-feedback",
-    "features/terms",
-    "features/messages-status", 
-    "features/security",
-    "features/email",
-    "features/sms",
-    "features/templates"
+    "contact",
+    "terms",
+    "message-status",
```
Comment (brdunfield) [inline a11y.json:10]: we probably want to leave this as messages-status unless we have direction to change it

File: app/templates/partials/privacy-en.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<h1 class="heading-large">Privacy</h1>
+<h1 class="heading-large">{{ _('Privacy notice: how we use your data') }}</h1>
```
Comment (brdunfield) [inline app/templates/partials/privacy-en.html:1]: I think this change should have been the other way around -> make the Title of the page match the existing h1

File: app/templates/partials/privacy-fr.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<h1 class="heading-large">Confidentialité</h1>
+<h1 class="heading-large">{{ _('Privacy notice: how we use your data') }}</h1>
```
Comment (brdunfield) [inline app/templates/partials/privacy-fr.html:1]: See comment on -en file

File: app/templates/partials/privacy-en.html (hunk 3 above)
Comment (AntoineAugusti) [reply app/templates/partials/privacy-en.html:1]: @anikbrazeau Can you confirm what you had in mind for this? https://github.com/cds-snc/notification-api/issues/979

File: a11y.json (hunk 2 above)
Comment (AntoineAugusti) [reply a11y.json]: There was a mix between singular/plural in method, routes and templates. Switched everything to plural.

File: app/templates/partials/privacy-en.html (hunk 3 above)
Comment (anikbrazeau) [reply app/templates/partials/privacy-en.html:1]: Yes, can we make the change the other way around 😄 
Make the H1 "Privacy" everywhere, and change the page title that appears in tabs to match "Privacy"
Sorry for the confusion.

File: app/templates/partials/privacy-fr.html (hunk 4 above)
Comment (anikbrazeau) [reply app/templates/partials/privacy-fr.html:1]: Ici, "Confidentialité" dans le titre de la page Web 😄

File: app/templates/views/cookies.html [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {% extends "withoutnav_template.html" %}
 
 {% block per_page_title %}
-  Cookies
+  {{ _('Cookies') }}
```
[the remark this answers is not in the document]
Comment (AntoineAugusti) [reply app/templates/views/cookies.html]: Just pushed a commit to delete the "cookies" page. We can always restore it later if it comes back 🍪

File: app/__init__.py [hunk 6]
```diff
@@ -496,7 +496,7 @@ def format_notification_status_as_field_status(status, notification_type):
 
 
 def format_notification_status_as_url(status, notification_type):
-    url = partial(url_for, "main.message_status")
+    url = partial(url_for, "main.messages_status")
```
Comment (brdunfield) [inline app/__init__.py:499]: In navigation.py this is still listed as message_status. Probably best to update navigation.py so we have some consistency here? I don't know how this inconsistency has lasted so long 😂

Comment (AntoineAugusti) [reply app/__init__.py:499]: It should not be, did you refresh the "Files changed" tab on GitHub? I've done it in https://github.com/cds-snc/notification-admin/pull/763/commits/3748214046bb4788d00557d43e1404c78af99681.

There are quite a few aggressive tests on `navigation.py` to catch these, you notice when endpoints are not properly named 🤣

Comment (brdunfield) [reply app/__init__.py:499]: Nevermind, I see its been updated there as well. Too many files 😂

**Comment by brdunfield** [comment]
So the "Set up a demo" form option does not appear to work in this PR (but does work in Staging). When you submit a form with that option, you get successfully redirected to the "Thanks" page, but the email does not go through and we do not get a FreshDesk notification in slack

**Comment by AntoineAugusti** [comment]
Seems like this was a false alarm. We both tested locally and on the review app and it seems fine. I added other subject types in the tests to make sure the form does not break with some values.

**Review by brdunfield — APPROVED** [review]
Agreed. Not sure why it did not work when I tested earlier!

Sorry for the delay on this one, it was very large and I needed to make sure I had enough time to do it justice.

👍 🦑
